tree/ntuple/v7/inc/ROOT/RCluster.hxx

```diff
@@ -45,7 +45,7 @@ private:
    /// The memory location of the bytes
    const void *fAddress = nullptr;
    /// The compressed and packed size of the page
-   std::size_t fSize = 0;
+   std::uint32_t fSize = 0;
```

Comment:
This is a potential significant change (even if the intent had always been to use only 32 bits).  I recommend calling it out explicitly in the commit log.

Reply:
Done